### DISCUSSION TITLE
[8.0] GuzzleHTTP update to 7.0+ necessary?

### DISCUSSION THREAD

**Discussion by SDekkers** [laravel/framework#34254] [General] [answered]
Other than a bump to 7+ in the laravel/laravel repository, I don't see much related that requires a change in the Guzzle package version.

I'm using cloudflare-sdk and some other packages which require guzzle 6. Would it be possible to use Laravel 8.0 with Guzzle 6?

**Comment by driesvints** [comment]
https://laravel.com/docs/8.x/upgrade#updating-dependencies

**Comment by browner12** [comment, marked as answer]
you should be free to require whichever version you need for your app, correct? If Laravel had a specific dependency on Guzzle, it would be listed in the `laravel/framework` dependencies.

I think this is more so they are encouraging you to move forward, but I don't think it would prevent the framework from working.
